point/verb_자고.yaml

```diff
@@ -30,7 +30,7 @@ details: |-
 
   # Shorthand {#shorthand}
 
-  For indirect quote:
+  # Shortened form {#shortened}
```

Comment:
Did you meant to change the line above ?

Reply:
Uhm I wanted to change Shorthand to # Shortened form {#shortened} to stay consistent